src/js/plugins/inheritance.js

```diff
@@ -994,9 +1002,16 @@
 
     updateInheritanceListInOpenedCardView: function (card) {
       if (card) {
-        var _descParentElement = document.querySelector('[attr="desc"]'),
+        // We want to grab the section of a Card which is right above description.
+        // addingMethod is used to call prependElement in helpers.js. 
```

Comment:
This may or may not be true, since the code may later change what's in `addingMethod` before it gets called.

Suggestion: `addingMethod is used to specify which helpers.js method to call`.

Reply:
Great point